### DISCUSSION TITLE
[Script request] Monica

### DISCUSSION THREAD

**Discussion by JcMinarro** [community-scripts/ProxmoxVE#24] [Request script]
### Application Name

Monica

### Website

https://www.monicahq.com/

### Description

Open source personal CRM.
Monica helps you organize the social interactions with your loved ones.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by MickLesk** [comment]
done
